test_utils/test_main.c

```diff
@@ -1605,8 +1605,8 @@ assertion_file_nlinks(const char *file, int line,
 	r = lstat(pathname, &st);
 	if (r == 0 && (int)st.st_nlink == nlinks)
 		return (1);
```

Comment:
In the _WIN32 ifdef right above this block you need instead of:
```c
failure_start(file, line, "File %s has %d links, expected %d",
    pathname, bhfi.nNumberOfLinks, nlinks);
```
the following code:
```c
failure_start(file, line, "File %s has %jd links, expected %d",
    pathname, (intmax_t)bhfi.nNumberOfLinks, nlinks);
```

Reply:
I think I got them all! Thanks!